plugin.py

```diff
@@ -158,18 +159,25 @@ def install_or_update(cls) -> None:
                 shutil.rmtree(cls.basedir())
             os.makedirs(cls.basedir(), exist_ok=True)
             version = cls.server_version()
-            url = URL.format(tag=TAG, arch=arch(), platform=platform())
-            gzipfile = os.path.join(cls.basedir(), "rust-analyzer.gz")
+            is_windows = sublime.platform() == "windows"
+            extension = "zip" if is_windows else "gz"
+            url = URL.format(tag=TAG, arch=arch(), platform=platform(), ext=extension)
+            gzipfile = os.path.join(cls.basedir(), f"rust-analyzer.{extension}")
             serverfile = os.path.join(
                 cls.basedir(),
-                "rust-analyzer.exe" if sublime.platform() == "windows" else "rust-analyzer"
+                "rust-analyzer.exe" if is_windows else "rust-analyzer"
             )
             with urllib.request.urlopen(url) as fp:
                 with open(gzipfile, "wb") as f:
                     f.write(fp.read())
-            with gzip.open(gzipfile, "rb") as fp:
-                with open(serverfile, "wb") as f:
-                    f.write(fp.read())
+
+            if is_windows:
+                with zipfile.ZipFile(gzipfile, "r") as zip_ref:
+                    zip_ref.extract("rust-analyzer.exe", cls.basedir())
```

Comment:
Can use the `gzipfile`/`archive_file` variable here.

Reply:
Sorry, got confused. But we could extract the `"rust-analyzer.exe" if is_windows else "rust-analyzer"` line into its own variable and reuse here.

Reply:
ah yes, I was wondering if I should do that. One moment